docs/docs/07-examples/14-wing-analytics.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Why is this under 07-examples?

Reply:
Just a placeholder for now, wanted to confirm where this was actually going to go.